### PR TITLE
Enable Kokoro buildbots.

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

**Comment by ehsannas** [comment]
PTAL @antiagainst @dneto0 
The MSCV-2017-Debug still fails. But it looks like the bots are running for PRs now. So If this change is merged, we should at least see all the bots passing except for 1.
I have filed an issue for MSVC-2017 version to be updated by Kokoro team. And I'll look into Docker.

**Comment by antiagainst** [comment]
Maybe we can disable the failing job for now and re-enable it later?

File: kokoro/android/build.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright (c) 2017 Google Inc.
```
Comment (antiagainst) [inline kokoro/android/build.sh]: 2018 Google LLC?

File: kokoro/android/build.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# Display commands being run.
+set -x
+
+BUILD_ROOT=$PWD
+SRC=$PWD/github/SPIRV-Tools
+TARGET_ARCH="armeabi-v7a with NEON"
+export ANDROID_NDK=/opt/android-ndk-r15c
+
+# Get NINJA.
+wget -q https://github.com/ninja-build/ninja/releases/download/v1.7.2/ninja-linux.zip
```
Comment (antiagainst) [inline kokoro/android/build.sh]: Kokoro does not have ninja installed by default?

The latest is v1.8.2, which is released on 2017-09-12: https://github.com/ninja-build/ninja/releases/tag/v1.8.2

Also for other places.

File: kokoro/ndk-build/build.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+BUILD_ROOT=$PWD
+SRC=$PWD/github/SPIRV-Tools
+
+# Set the glslang repo URL.
+GLSLANG_REPO_URL="https://github.com/KhronosGroup/glslang.git"
```
Comment (antiagainst) [inline kokoro/ndk-build/build.sh]: We do not need Glslang for SPIRV-Tools?

File: kokoro/android/build.sh (hunk 2 above)
Comment (ehsannas) [reply kokoro/android/build.sh]: Done

File: kokoro/ndk-build/build.sh (hunk 3 above)
Comment (ehsannas) [reply kokoro/ndk-build/build.sh]: Oops. Copy-pasta from shaderc... Done

**Comment by antiagainst** [comment]
+1. I'd suggest we disable MSCV-2017-Debug for now and get this submitted. How do you think @dneto0?

**Comment by dneto0** [comment]
> +1. I'd suggest we disable MSCV-2017-Debug for now and get this submitted. How do you think @dneto0?

I'd still want coverage of that case.  Maybe keep the AppVeyor bot *only* for MSVC-2017-debug and enable Kokoro for everything else.

**Comment by ehsannas** [comment]
Yeah we don't want to lose coverage. I am not removing the existing bots until the new ones are all stable and green.

**Pull request merged**